tests/models/test_abstract.py

```diff
@@ -120,8 +129,19 @@ def test_time_step(bmi, model: MockedModel):
 
 
 def test_output_var_names(bmi, model: MockedModel):
-    bmi.get_output_var_names.return_value = ('discharge',)
+    bmi.get_output_var_names.return_value = ('discharge', )
 
     names = model.output_var_names
 
-    assert names == ('discharge',)
+    assert names == ('discharge', )
+
+
+def test_get_value_as_xarray(model: MockedModel):
+    expected = np.array([[1.0, 2.0]])
+
+    dataarray = model.get_value_as_xarray("Temperature")
+
+    assert_array_equal(dataarray.values, expected)
+    assert dataarray.name == "Temperature"
+    assert dataarray.units == "degC"
+    assert isinstance(dataarray, xr.DataArray)
```

Comment:
Could use http://xarray.pydata.org/en/stable/generated/xarray.testing.assert_allclose.html#xarray-testing-assert-allclose instead of multiple asserts.

Reply:
Is there a reason you chose `allcose` instead of `equal`? I noticed you used that for the numpy arrays as well.

Reply:
Any computation could lead to slightly different floats on different machines. To prevent tests from failing when run on another machine I always use allclose. I use allclose all the time so I dont have to think about what the computation is and how reproducible it is.

Reply:
In this case the test just compares mocked against expected. No computation happens which could change the floats. So here using equal is fine.